src/compounds/billboard/src/index.tsx

```diff
@@ -112,15 +116,22 @@ const Billboard: React.FC<Props> = ({
                   {React.cloneElement(primarySlot, { variant: 'billboard' })}
                 </div>
               )}
-
               {!primarySlot && bgImage && (
                 <div
                   sx={{
-                    width: '404px',
-                    height: '416px',
-                    display: ['none', 'none', 'block'],
-                    background: `url(${bgImage}) no-repeat right bottom / contain`,
-                    variant: 'compounds.billboard.bgImagePosition'
+                    width: ['90%', '404px', '404px'],
+                    height: ['0', '255px', '416px'],
+                    margin: ['30px auto 0', '30px auto 0', '0'],
+                    paddingTop: ['60%', '0', '0'],
+                    display: [
+                      showImageOnMobile ? 'block' : 'none',
+                      showImageOnMobile ? 'block' : 'none',
+                      'block'
+                    ],
+                    background: `url(${bgImage}) no-repeat ${imagePosition} / contain`,
+                    variant: !imagePosition
```

Comment:
If I understood this correctly, doesn't this mean that for all available components in contentful, variant will now be empty (bc imagePosition will be `right bottom` by default and ! will make it `false`) whereas it should be `compounds.billboard.bgImagePosition`?